src/internal/styles/forms/mixins.scss

```diff
@@ -147,7 +147,6 @@
     @include utils.base-pseudo-element;
```

Comment:
Since this is not a shadow anymore, I would give a try at making this an actual outline, and it looks like actually this enter `&::after` block can be replaced with
```
outline: 1px solid awsui.$color-grey-350;
```
and we can probably remove the specific shadow styling for the stacked container in `src/container/styles.scss` as well as the box shadow definitions in `style-dictionary/visual-refresh/shadows.ts`.

I would also rename the mixin from `container-shadow` to `container-border`.